CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project does NOT adhere to [Semantic Versioning](https://semver.org/spec/v2
 * `[sitecore-jss-angular][templates/angular]` jss-angular package and sample has been updated to version 14. This means several changes:
   * JSS Angular sample is now using Ivy
   * IE11 no longer supported by JSS Angular
+  * You can no longer specify lazy-loaded routes by setting a string value to loadChildren. Make sure you move to dynamic ESM import statements.
```

Comment:
Please, provide an example of how to make this migration

Reply:
Done. Please check if it will suffice.